src/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: AvenirBook;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  outline: none;
+  padding: 20px 62px;
```

Comment:
Use only left padding and line-height for aligning text by center (should fix tests)